content/docs/hello-world.md

```diff
@@ -15,36 +15,43 @@ ReactDOM.render(
 );
 ```
 
-It displays a heading saying "Hello, world!" on the page.
+Het weergeeft een kop met de tekst "Hello, world!" op de pagina.
 
-[](codepen://hello-world)
+[Probeer het op CodePen](codepen://hello-world)
 
-Click the link above to open an online editor. Feel free to make some changes, and see how they affect the output. Most pages in this guide will have editable examples like this one.
+Klik op de bovenstaande link om een online editor te openen.
+Voel je vrij om wat aanpassingen te maken en te zien hoe deze de uitvoer beïnvloeden.
+De meeste pagina's in deze gids hebben dit soort bewerkbare voorbeelden.
 
 
-## How to Read This Guide {#how-to-read-this-guide}
+## Hoe Lees Je Deze Gids {#how-to-read-this-guide}
 
-In this guide, we will examine the building blocks of React apps: elements and components. Once you master them, you can create complex apps from small reusable pieces.
+In deze gids zullen we de bouwstenen van React-applicaties onderzoeken: elementen en componenten.
+Van zodra je ze onder de knie hebt, kan je complexe applicaties ontwikkelen uit kleine, herbruikbare stukjes.
 
 >Tip
 >
->This guide is designed for people who prefer **learning concepts step by step**. If you prefer to learn by doing, check out our [practical tutorial](/tutorial/tutorial.html). You might find this guide and the tutorial complementary to each other.
+>Deze gids is bedoeld voor zij die liever **concepten stap voor stap leren**. Als je liever al doende leert, bekijk dan onze [praktische tutorial](/tutorial/tutorial.html). Mogelijk vind je dan dat deze gids en de tutorial elkaar aanvullen.
```

Comment:
```suggestion
>Deze gids is bedoeld voor mensen die liever **concepten stap voor stap leren**. Als je liever al doende leert, bekijk dan onze [praktische tutorial](/tutorial/tutorial.html). Je zult dan zien dat deze gids en de tutorial elkaar aanvullen.
```

Reply:
@Keraito I would opt for "je" for this kind of a text, definitely considering the potentially younger audience.

Reply:
Agreed, @jochenboesmans. Using "je" seems to fit better with our target audience.